SamplesDashboard/SamplesDashboard/ClientApp/src/views/details/Details.tsx

```diff
@@ -70,8 +70,9 @@ export default class Details extends React.Component<any, any> {
         );
         const data = await response.json();
         if (data.dependencyGraphManifests.nodes[0]) {
-            for (let index = 0; index < data.dependencyGraphManifests.nodes.length; index++) {
-                data.dependencyGraphManifests.nodes[index].dependencies.nodes.forEach((element: any) => this.allItems.push(element));
+            for ( const index of data.dependencyGraphManifests.nodes) {
```

Comment:
Changing this loop breaks the intention and functionality of the code. Please revisit